docs/changelog/97463.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 97463
+summary: "[Enterprise Search] Add connectors indices and ent-search pipeline"
+area: Application
```

Comment:
again, not sure it should be `Application` here.
because that implies any subsequent changes we make for connectors, all be in the `Application` area, which is not the case, right?

Reply:
@ioanatia If that's the case should we consider renaming the ent search module from `Application` to something broader?

Reply:
looking at other changelog entries in the `Application` area, I guess it's fine for now to use `Application` here